src/classes/AirPlane.java

```diff
@@ -5,34 +5,30 @@
 public class AirPlane implements BookSeat {
     private String name;
     private String modelNumber;
-    private int maxSeats;
+    private int capacity;
     private int rows;
+    private int cols;
     private int bookedSeats;
     private char[][] seats;
 
-    public AirPlane(String planeName, String model, int rows) {
+    public AirPlane(String planeName, String model, int rows, int cols) {
         this.name = planeName;
         this.modelNumber = model;
         this.rows = rows;
-        this.maxSeats = 10 * rows;
+        this.cols=cols;
+        this.capacity = cols * rows;
         setSeats();
     }
 
     public void setSeats() {
-        this.seats = new char[this.rows][10];
-        for (int i = 0; i < this.rows; i++) {
-            seats[i][0] = 'A';
-            seats[i][1] = 'B';
-            seats[i][2] = 'C';
-            seats[i][3] = 'D';
-            seats[i][4] = 'E';
-            seats[i][5] = 'F';
-            seats[i][6] = 'G';
-            seats[i][7] = 'H';
-            seats[i][8] = 'I';
-            seats[i][9] = 'J';
+        this.seats = new char[this.rows][this.cols];
+        int firstCol = (int) 'A';
+        System.out.println(firstCol);
```

Comment:
Do we need this?

Reply:
I forgot to remove the print statement but the firstCol one needed that value inside the for loop
`seats[r][c]= (char) ( firstCol + c);`